### DISCUSSION TITLE
Scroll float window line by line

### DISCUSSION THREAD

**Discussion by ricardoseriani** [neoclide/coc.nvim#5068] [Q&A] [answered]
There are any way that I can scroll the float window line by line?

I thought that I can pass the number of lines to scroll to the function `coc#float#scroll(<number>)` but it seems that the following map seems to be scrolling by "page":

```
  nnoremap <silent><nowait><expr> <C-f> coc#float#has_scroll() ? coc#float#scroll(1) : "<leader><C-f>"
  nnoremap <silent><nowait><expr> <C-b> coc#float#has_scroll() ? coc#float#scroll(0) : "<leader><C-b>"
```

**Comment by fannheyward** [comment, marked as answer]
~No such feature yet. ~

`coc#float#scroll(1, 1)`, check `:h coc#float#scroll`.

**Reply by ricardoseriani** [reply]
Oh, I understand.

Thanks for your answer.

There are any plans for adding something like this?

**Reply by hexh250786313** [reply]
@fannheyward @ricardoseriani 

Doesn't it really support? Try:

```vim
coc#float#scroll(1, 1)
coc#float#scroll(0, 1)
```

**Reply by fannheyward** [reply]
@hexh250786313 you're right, the second parameter is scroll line number.

**Reply by ricardoseriani** [reply]
Awesome!

That worked like a charm.

Thanks for your help @hexh250786313.
